gen.py

```diff
@@ -104,25 +133,69 @@ def schema_extra(schema: Dict[str, Any], _) -> None:
             schema["maxProperties"] = 1
 
 
+class VarPath(BaseModel):
+    __root__: str = Field(
+            ..., description="Path to params file for parametrization."
+    )
+
+
 class VarDecl(BaseModel):
     # {"foo" (str) : "foobar" (Any) }
-    __root__: Dict[VarKey, Any]
+    __root__: Dict[VarKey, Any] = Field(
+            ..., description="Mapping for parametrization."
+    )
 
 
 class Vars(BaseModel):
-    __root__: List[Union[FilePath, VarDecl]]
+    __root__: List[Union[VarPath, VarDecl]]
+
+
+STAGE_VARS_DESC = """\
+Sequence of variables for parametrization.
+
+May include any mapping or a path to a params file (which may be a string or a \
+mapping to params in the file).
+
+Use in the stage with the `${}` substitution expression."""
+
+CMD_DESC = """\
+(Required) Command to run (anything your system terminal can run).
+
+Can be a string or a sequence of commands."""
```

Comment:
Why the copy edits `your` -> `the` and `can run` -> `will accept`? This is taken from https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#stage-commands.

Reply:
your->the  because we don't know where the stage will run. Could be on CI or someone else's env.
can run->will accept  reads better ("run" is already in the sentence).